core/lib/Thelia/Config/I18n/en_US.php

```diff
@@ -55,7 +55,7 @@
     'Associate documents' => 'Associate documents',
     'Associate images' => 'Associate images',
     'At least one of cart products categories is %op% <strong>%categories_list%</strong>' => 'At least one of cart products categories is %op% <strong>%categories_list%</strong>',
-    'Attribut' => 'Attribut',
+    'Attribut' => 'Attribute',
```

Comment:
It would be great to change the key too, in this file and where this key is used in translate functions.